### DISCUSSION TITLE
How do I configure the day length when setting estimates?

### DISCUSSION THREAD

**Discussion by unikitty37** [super-productivity/super-productivity#3245] [Q&A] [answered]
If I enter "1d" when setting an estimate for a task, it is entered as 24h. But I don't work every hour of the day — nobody does! I track time against 7 hour days, so I would like to set a day length option accordingly: when I enter "1d", "2d", or "3d" it's treated the same as entering "7h", "14h", or "21h". This would bring superProductivity into line with GitLab and ClickUp, which allow this to be configured.

Computers are good at multiplication, so I'd much rather the computer did that work rather than me having to do it 😁 

Is there already a way to configure this that I've missed? I couldn't see anything in Settings…

**Comment by johannesjo** [comment, marked as answer]
Thanks for the suggestion. I am all for it! Could you please create an issue fro this?

**Reply by unikitty37** [reply]
Done — thanks!

**Reply by johannesjo** [reply]
Thank you!!
